web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1841,6 +1841,15 @@
                   : "";
             return angular.isFunction(this[fnName]) ? this[fnName]() : false;
           },
+          canViewMetadataHistory: function () {
+            var profile = gnConfig["metadata.history.accesslevel"] || 'Editor',
+              fnName =
+                profile !== ''
+                  ? 'is' + profile[0].toUpperCase() + profile.substring(1) + 'OrMore'
+                  : '';
+            console.log("User profile to view history: "+profile);
+            return angular.isFunction(this[fnName]) ? this[fnName]() : false;
```

Comment:
I tested with registered user and did not see history.  Did you test with registered user?

Reply:
Yes, you may need to flush your browser cache.

![image](https://github.com/geonetwork/core-geonetwork/assets/74916635/9aaf3872-6ff6-468c-8473-1122f1553729)


![image](https://github.com/geonetwork/core-geonetwork/assets/74916635/7bb1d796-c41b-4889-abc9-8f1a0fcf30f9)

Reply:
@ianwallen 

I have retested it and I think the issue is original user who can see history is based on this logic. They must be an editor and this person needs to have authority to edit this record. Since we are introducing registered user to view history page and they are totally has no authority to edit any record. Do we want to change this logic?

https://github.com/geonetwork/core-geonetwork/blob/f6caf39363b73a3712a6c862f6ac46541e5364c8/services/src/main/java/org/fao/geonet/api/records/MetadataWorkflowApi.java#L698-L721